python/ray/tests/test_autoscaler_fake_multinode.py

```diff
@@ -100,6 +101,53 @@ def i():
     # __example_end__
 
 
+@pytest.mark.parametrize("autoscaler_v2", [False], ids=["v1"])
+@pytest.mark.parametrize("placement_strategy", ["SPREAD", "STRICT_SPREAD"])
+def test_fake_autoscaler_placement_group_partial_idle(
+    autoscaler_v2, placement_strategy
+):  # Test the cases of start_node_delay_s >= idle_timeout_minutes.
```

Comment:
Please write the details instead of pointing to the PR.

Reply:
👍